lib/strtoofft.c

```diff
@@ -225,7 +225,7 @@ CURLofft curlx_strtoofft(const char *str, char **endp, int base,
 
   while(*str && ISBLANK(*str))
     str++;
-  if(('-' == *str) || (ISSPACE(*str))) {
+  if(('-' == *str) || (ISSPACEBASE(*str))) {
```

Comment:
I would rather suggest:
~~~c
if(!ISXDIGIT(x))
 ...

Reply:
done

Reply:
Instead of `if(('-' == *str) || (ISSPACEBASE(*str)))`?

Reply:
Or maybe just leave it? This `ISSPACEBASE` does not increase readability.